types/stripe-js/embedded-checkout.d.ts

```diff
@@ -1,3 +1,26 @@
+export type StripeEmbeddedCheckoutAddress = {
+  country: string;
+  line1?: string | null;
```

Comment:
non-blocking: I think we can guarantee line1 is present so we could update this typing to be more strict

Reply:
I just copied type from our internal code. We can update it once we change the internal type.